README.md

```diff
@@ -26,7 +28,7 @@ What does the following selector do?  `ul.dropdown > li`?
 Select 1:
 ```
 [] Selects all li's which are directly inside a ul of class dropdown (children)
-[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[X] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
 [] Selects all ul's of class dropdown, as well as the children elements that are li's
 [] Selects all ul's of class dropdown, only if their children are exclusively li's
 ```
```

Comment:
Correct answer is the first one. `>` means you are selecting the "direct children" (i.e., one level down) of an element.